modules/view-kubernetes-secrets.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="view-kubernetes-secrets_{context}"]
+= Viewing Kubernetes secrets
+
+[role="_abstract"]
+View Kubernetes secrets in use in your environment and identify deployments using those secrets.
+
+.Procedure
+. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
+. On the *Secrets Most Used Across Deployments* widget, select *View All*.
+Alternatively, select *Application & Infrastructure* -> *Secrets* from the *Configuration Management* view header.
```

Comment:
You could consider only including one of the methods if both routes navigate you to the same view, to minimise the step.